app/src/androidTest/java/com/github/epfl/meili/home/MockAuthenticationService.kt

```diff
@@ -0,0 +1,34 @@
+package com.github.epfl.meili.home
+
+import android.app.Activity
+import android.content.Intent
+
+class MockAuthenticationService : AuthenticationService {
+    var MOCK_NAME = "Meili User"
```

Comment:
I think this should be a private const val, and be in a companion object.

Reply:
I will change this! Do you know what's the policy for using companion objects? Like, when should we be using them?

Reply:
Actually, in this case, I was modifying the values in the test. So they weren't constants, I removed the capital letters which were indeed misleading